README.md

```diff
@@ -43,4 +43,87 @@ If you'd like to try out the plugin pre-release, you can build it from source an
 
 ## Getting started
 
+The plugin can work in any of the  [JetBrains family of IDEs](https://www.jetbrains.com/products.html), let’s try the first example with Kubernetes and Java in IntelliJ IDE (the plugin supports both Community and Ultimate editions). Follow the installation steps above to install the plugin. Restart your IDE if prompted to activate the plugin.
+
+Clone the repository to your local machine to get your copy of the repository:
+```
+git clone https://github.com/GoogleContainerTools/google-container-tools-intellij.git
+```
+
+Open the `hello-spring-boot` example from `google-container-tools/examples` directory with your IntelliJ IDE. You can either point to the directory or to the Maven build file (`pom.xml`). The project opens and loads:
+
+<img src="docs/images/sb-hello-world-project.png" alt="opened Spring Boot hello world project" width="500"/> 
+
+This project is a very simple web application created with [the popular Spring Boot framework](https://spring.io/projects/spring-boot). It uses [Jib Maven plugin](https://github.com/GoogleContainerTools/jib) to build a container image for the project, instead of a Dockerfile. Jib is supported by the plugin and Skaffold and recommended as the best way to build optimized Java container images.
+
+Once the project loads, the plugin will detect it is preconfigured with Skaffold YAML configuration to build image and deploy to Kubernetes. A notification shows:
```

Comment:
nits:

"will detect it is preconfigured" -> "will detect the Skaffold configuration and and prompt to create the Kubernetes targets. The notification shows:"

Reply:
sounds clean, changed, thanks.